src/clj/game/utils.clj

```diff
@@ -1,5 +1,6 @@
 (ns game.utils
   (:require
+    [game.core.state]
```

Comment:
Don't do this. This namespace is meant to be fully independent from the engine. Given that we're only referring to swap!* in engine or cards functions, there's no need to check if the given atom is an instant of State.